src/openms/source/ANALYSIS/ID/ConsensusMapMergerAlgorithm.cpp

```diff
@@ -377,9 +376,13 @@ namespace OpenMS
         break;
       }
       const ProteinIdentification::SearchParameters& sp = idRun.getSearchParameters();
+      String spdb = sp.db;
```

Comment:
seeing this code duplicated 4x and the code below (which looks like a tolerant version of `operator==` for ProteinIdentification::SearchParameters): can we please move this whole comparison to a member of ProteinIdentification::SearchParameters?!

Reply:
true. will do.